### DISCUSSION TITLE
No module named 'py_eddy_tracker_sample'

### DISCUSSION THREAD

**Discussion by rampatels** [AntSimi/py-eddy-tracker#166] [General]
I followed the instruction given in README.md file of the project to set up this toolbox. I confess that I am new to this python world. But has basic understanding of setting up conda environment and all.

# How I tried to set up  
# creating a working directory
$ mkdir ~/pyeddytracker
$ cd pyeddytracker
$ python3 -m venv metaeddy
$ cd metaeddy/
$ sudo apt install python3-pip
$ pip install numpy scipy netCDF4 matplotlib opencv-python pyyaml pint polygon3
$ git clone https://github.com/AntSimi/py-eddy-tracker
$ cd py-eddy-tracker/
$  sudo python3 setup.py install
$ jlab 
this opens in my current working directory. Then I move to notebooks directory and try any example it prompts that no py_eddy_tracker_sample. 

Any help would be appreaciated.

**Comment by AntSimi** [comment]
You don't need sudo but you need to activate your venv

**Comment by AntSimi** [comment]
I read too quickly, do pip install pyeddytrackersample
